src/main/java/TwitterApplication.java

```diff
@@ -1,3 +1,4 @@
+import configuration.ApplicationConfiguration;
```

Comment:
unused import?

Reply:
This is used when TwitterApplication extends Application<ApplicationConfiguration> so that it can implement run

Reply:
Do we still need the `ApplicationConfiguration` class?